CONTRIBUTING.md

```diff
@@ -92,6 +93,52 @@ Your PR can contain more than one patch. Use as many patches as necessary to
 implement the request. Each PR should only cover one topic. If you mix up
 different items in your patches or PR, they will likely need to be reworked.
 
+### Assisted PR workflow
+
+If your PR is deemed useful but you are struggling to update it based on
+review feedback, or your PR appears to have been abandoned, the project
+maintainers may suggest (or decide in your absence) that an assisted PR
+workflow be employed. This involves creating a new PR using your PR as a base
+and applying fixes to it.
+
+The full workflow:
+
+1. A team member will create a new PR with the same content as the original
+   one.
+
```

Comment:
Can we have a step somewhere that notes the team member will also mark the old PR as DNM, and add a reference comment to the new PR.